src/transformers/generation/configuration_utils.py

```diff
@@ -139,8 +138,6 @@ class GenerationConfig(PushToHubMixin):
         num_beam_groups (`int`, *optional*, defaults to 1):
             Number of groups to divide `num_beams` into in order to ensure diversity among different groups of beams.
             [this paper](https://huggingface.co/papers/1610.02424) for more details.
-        penalty_alpha (`float`, *optional*):
```

Comment:
In `__init__` below, when we pop things from `kwargs`, let's create a `# Deprecated/Moved to Hub` section and place these arguments there (let's do the same for the DoLa args)